### ISSUE TITLE
Bug in co-ordinate search feature

### ISSUE THREAD

**Issue by Vonter** [Vonter/blr-boundaries#27]
The co-ordinate search feature implemented in https://github.com/Vonter/blr-boundaries/pull/25 has a bug where the search returns 0 matching boundaries. The bug appears intermittently.

**Comment by thedivtagguy** [comment]
In `Map.svelte`, you're doing this: 

```js
      $mapStore
        .addSource('boundaries', {
          type: 'geojson',
          promoteId: 'namecol',
          data: $boundaries
        })
        .addSource('boundaries-centerpoints', {
          type: 'geojson',
          data: {
            type: 'FeatureCollection',
            features: $boundaries.features.map((feature: Feature) => {
              feature.geometry = {
                type: 'Point',
                coordinates: findPolylabel(feature)
              };
              return feature;
            })
          }
        });
        
 ```
 
 This converts boundaries to points, which is then what you're trying to access inside CoordinateDetails and pass to `polygon-lookup`. Except these are no longer polygons so nothing is going to work. 
 
 I've addressed this with a fix in 3261859fdc4aa76b1caf464d6cc3c2585a6fa669

**Comment by Vonter** [comment]
Thanks for investigating this bug @thedivtagguy. I didn't realize that `$boundaries` was being overwritten there, good find!
